### ISSUE TITLE
Nested headings

### ISSUE THREAD

**Issue by rmrlangford** [HL7/ig-template-base#210]
**Given**
1. Add heading <h3>Categories</h3> <a name="vaccine-status-categories"> </a>
2. Add paragraph
3. Add sub-heading <h4>Unvaccinated </h4>
4. Add paragraph
5. Add another sub-heading <h4>Partially Vaccinated </h4>
6. Add paragraph

**Actual Behaviour**
Outcome:
1.1 Categories
some text
1.1.1 Unvaccinated
some text
1.1.1 Partially Vaccinated 
some text

**Expected Behaviour**
Outcome:
1.1 Categories
some text
1.1.1 Unvaccinated
some text
1.1.2 Partially Vaccinated 
some text

In summary, the numbers for sub-headings are not auto-incrementing.

**Comment by rmrlangford** [comment]
FYI @costateixeira

**Comment by lmckenzi** [comment]
@rmrlangford Where exactly are you doing this?  Are you starting with level-3 headings?

**Comment by rmrlangford** [comment]
Hi, honestly can't remember, been too long since the creation of this ticket. This is no longer an issue though and can be closed.
